playbooks/02 - Use Case - CICD/Setup CICD Environment.json

```diff
@@ -17,6 +17,29 @@
     "versions": [],
     "triggerStep": "\/api\/3\/workflow_steps\/e3243c1a-daf7-4bf1-901a-8cc380e797f2",
     "steps": [
+        {
+            "@type": "WorkflowStep",
+            "name": "Activate Source Control Sync Status Schedule",
+            "description": null,
+            "arguments": {
+                "params": {
+                    "iri": "\/api\/wf\/api\/scheduled\/{{vars.steps.Get_Source_Control_Sync_Status_Schedule_Details.data['hydra:member'][0].id}}\/?format=json",
+                    "body": "{\n    \"id\": \"gAAAAABogL_f0z1Br_2VEIMqyq7kbg6BxCcPjgUkhluoH06hO1Zq-xwD97YR0rdjwJz_tGUS6YiT9_mFeJeS9JnOHcH_mFM5uQ==\",\n    \"crontab\": {\n        \"id\": 3,\n        \"minute\": \"0\",\n        \"hour\": \"*\/1\",\n        \"day_of_week\": \"*\",\n        \"day_of_month\": \"*\",\n        \"month_of_year\": \"*\",\n        \"timezone\": \"UTC\"\n    },\n    \"interval\": null,\n    \"name\": \"Ingest_Source-Control-Sync-Status(CICD)\",\n    \"task\": \"workflow.tasks.periodic_task\",\n    \"enabled\": true\n}",
```

Comment:
we can we have schedule configurable in set variable step for easy editing. 
Suggestion: Some desc in the PB step to elloborate the its working will help.

Reply:
not required customer can update it by updating schedule